assets/css/general.css

```diff
@@ -26,6 +26,32 @@ h2 {
     z-index: 10000;
 }
 
+/* sass-lint:disable class-name-format*/
+.position-relative {
+    position: relative;
+}
+
+.d-none {
+    display: none;
+}
+
+.d-inline-block {
+    display: inline-block;
+}
+
+.text-center {
+    text-align: center;
+}
+
+.position-absolute {
+    position: absolute;
+}
+
+.font-weight-bold {
+    font-weight: bold;
+}
+
```

Comment:
Do you mind removing this line and adding one after the comment instead? Also, there should be spacing at the end of the comments, like `class-name-format */`.